### PR TITLE
Added Licenses and About

### PR DESCRIPTION
**Pull request by WestonHanners**
# Checklist
- [x] I have read [CONTRIBUTING.md](./CONTRIBUTING.md)
- [x] I have described what this PR contains
- [x] This PR addresses one or more open issues that were assigned to me:
      - *list issue(s) here*
- [x] If this PR alters the UI, I have attached pictures/videos

# Pull Request Information

## About this Pull Request
Includes license notices for OSS software
Includes new About screen.

## Screenshots and Videos
image
IMG_1171

### CODE REVIEW

**Review by EricBAndrews — APPROVED** [review]
LGTM

**Comment by JakeShirley** [comment]
:shipit:

File: Mlem/Views/Tabs/Settings/About.swift [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        Group {
+            List {
+                Section {
+                    DeveloperView(name: "Weston",
+                                  link: "https://techhub.social/@weston")
+                    DeveloperView(name: "Jake",
+                                  link: "https://github.com/JakeShirley")
+                    DeveloperView(name: "Eric",
+                                  link: "https://github.com/EricBAndrews")
+                    DeveloperView(name: "Mormaer",
```
[the remark this answers is not in the document]
Comment (WestonHanners) [reply Mlem/Views/Tabs/Settings/About.swift:44]: I can do that. No pressure.

File: Mlem.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -544,14 +541,10 @@
...
-		CD1446282A5CAAAD00610EF1 /* Plain Vote Complex.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = "Plain Vote Complex.swift"; sourceTree = "<group>"; };
 		CD1446202A5B328E00610EF1 /* Privacy Policy.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = "Privacy Policy.swift"; sourceTree = "<group>"; };
 		CD1446222A5B336900610EF1 /* Documents View.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = "Documents View.swift"; sourceTree = "<group>"; };
 		CD1446242A5B357900610EF1 /* Document.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = Document.swift; sourceTree = "<group>"; };
 		CD1446262A5B36DA00610EF1 /* EULA.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = EULA.swift; sourceTree = "<group>"; };
-		CD1446172A58FC3B00610EF1 /* InfoStack.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = InfoStack.swift; sourceTree = "<group>"; };
```
[the remark this answers is not in the document]
Comment (WestonHanners) [reply Mlem.xcodeproj/project.pbxproj:552]: I think so because every time I remove these changes and rebuild it removes these files again.

**Comment by WestonHanners** [comment]
> @WestonHanners #283 is removing the cached async image library so an update will be needed for that

I'll do that after that PR is merged, not 100% sure its going into 1.0 yet.

**Pull request merged**